website/blog/2021-06-24-introduce-superbench.md

```diff
@@ -8,4 +8,113 @@ author_url: https://github.com/TobeyQin
 tags: [superbench, announcement]
 ---
 
-We are very happy to introduce [SuperBench](https://github.com/microsoft/superbenchmark) to help you validate AI infrastructure.
+
+This blog is to introduce [SuperBench](https://github.com/microsoft/superbenchmark) to help you validate AI infrastructure.
+
+We designed and developed SuperBench as a avalidation and profiling tool for AI infrastructure. 
+We try our best to create a modular, easy-to-use, standardized, and extensible tool. 
+Help users to complete and accurate assessment of AI infrastructure, and bring good user experience.
+
+
+## The Advantages of SuperBench
+
+### Easy-to-use CLI
+
+In order to achieve one-click deployment, one-click user experience, SuperBench provides a command line interface to help users deploy and run benchmarks. 
+Empowered by SuperBench CLI, user can deploy and run their benchmarks with only one command, which greatly shorten the learning time of using tools, 
+help user easily evaluate the performance of AI workload.
+
+Below is a simple example to show how to deploy and run benchmarks locally. For more information, 
+please view [CLI Document](https://github.com/microsoft/superbenchmark/blob/main/docs/cli.md)
+
+1. Deploy
+
+    ```bash
+    sb deploy -f local.ini
+    ```
+2. Run Benchmark
+
+    ```bash
+    sb run -f local.ini -c config.yaml
+    ```
+
+Among them, `local.ini` is the configuration file to manage nodes. If you want to run benchmarks locally, the node is `localhost`.
+
+```ini title="local.ini"
+[all]
+localhost ansible_connection=local
+```
+`config.yaml` is a config file to configure the details of benchmarkings. You can customize your benchmarks by modified this file.
+
+For more information, please view [confiuration](https://microsoft.github.io/superbenchmark/docs/getting-started/configuration)
+
+### Modular and Extensible Framework
+
+1. Executor Framework
+
+In order to facilitate the benchmarking and validation of large-scale clusters, we designed and implemented a modular and extensible framework. 
+SuperBench framework includes a runner as control node, as well as multiple executors as worker nodes. 
+A runner received commands from CLI and distribute to all nodes (worker nodes) in the cluster and collect results. 
```

Comment:
`collect data, and summarize the results on runner.`

Reply:
changed